Dockerfile

```diff
@@ -60,32 +60,30 @@ RUN dpkg --add-architecture arm64 && \
   apt-get install -y crossbuild-essential-amd64 crossbuild-essential-arm64 git libbtrfs-dev:amd64 libbtrfs-dev:arm64 libseccomp-dev:amd64 libseccomp-dev:arm64
 
 FROM build-base-debian AS build-containerd
+ARG TARGETARCH
 ARG CONTAINERD_VERSION
 RUN git clone https://github.com/containerd/containerd.git /go/src/github.com/containerd/containerd
+RUN TARGETARCH=${TARGETARCH:-amd64}
```

Comment:
wrong

Reply:
Hmm! I did not understand this at a lot of places here; we default to amd64 if not present. 
What do you mean?